src/main/java/seedu/eduke8/ui/Ui.java

```diff
@@ -503,7 +512,9 @@ private String listOfBookmarkedQuestions(BookmarkList bookmarks) {
             ArrayList<Displayable> optionsAvailable = properQuestion.getOptionList().getInnerList();
             int j = 1;
             for (Displayable option : optionsAvailable) {
+                Option castedOption = (Option) option;
                 optionOutput += System.lineSeparator() + "    " + j + CLOSE_BRACKET + option.getDescription()
+                        + (castedOption.isCorrectAnswer() ? MESSAGE_BOOKMARK_CORRECT_ANS_INDICATOR : "")
```

Comment:
Perhaps a constant could be used in place of `""`

Reply:
This one I'm not too sure, I noticed other classes with blank strings as well, figured we were leaving empty strings as empty strings.